### PR TITLE
Add fundraising thermometer to fundraising post

### PR DESCRIPTION
**Pull request by Discordius**
┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1208926371362063) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/common/FundraisingThermometer.tsx [hunk 1]
```diff
@@ -165,15 +177,15 @@ const FundraisingThermometer: React.FC<FundraisingThermometerProps & {classes: C
   const { LWTooltip } = Components
 
   return (
-    <div className={classes.root}>
+    <div className={classNames(classes.root, onPost && classes.onPost)}>
```
Comment (rjmk) [inline packages/lesswrong/components/common/FundraisingThermometer.tsx:180]: I think @b0b3rt wants you to do this a different way

Like `{ [classes.onPost]: onPost }` or sthg?

Comment (b0b3rt) [reply packages/lesswrong/components/common/FundraisingThermometer.tsx:180]: I forgot that we've moved on to this style (since it's more performant)

Comment (Discordius) [reply packages/lesswrong/components/common/FundraisingThermometer.tsx:180]: K, I'll leave it as is

File: packages/lesswrong/components/posts/PostsPage/PostsPage.tsx [hunk 2]
```diff
@@ -804,6 +804,8 @@ const { HeadTags, CitationTags, PostsPagePostHeader, LWPostsPageHeader, PostsPag
             toggleEmbeddedPlayer={toggleEmbeddedPlayer}
             dialogueResponses={debateResponses} 
             annualReviewMarketInfo={marketInfo}/>}
+          {lightconeFundraiserActive.get() && (post._id === '5n2ZQcbc7r4R8mvqc') &&
```
Comment (rjmk) [inline packages/lesswrong/components/posts/PostsPage/PostsPage.tsx:807]: I would prefer using the DB setting here

**Review by rjmk — APPROVED** [review]

**Pull request merged**